### DISCUSSION TITLE
请问提供的exe最新版能去掉命令行窗口吗

### DISCUSSION THREAD

**Discussion by xiaohaibao** [taojy123/KeymouseGo#50] [General]
打开就弹出一个命令行窗口，然后再出现图形界面，把命令行窗口关了应用就跟着关掉了，体验不太好，就最新版的3.2.1版本，暂时退回3.0版本了

**Comment by taojy123** [comment]
打包时忘记去掉命令行窗口了 
我又打了个包，可以重新下载最新版本
